opencti-platform/opencti-graphql/tests/02-integration/01-database/elasticSearch-test.js

```diff
@@ -404,7 +404,7 @@ describe('Elasticsearch pagination', () => {
   it('should entity paginate everything', async () => {
     const data = await elPaginate(testContext, ADMIN_USER, READ_ENTITIES_INDICES, { first: ES_MAX_PAGINATION });
     expect(data).not.toBeNull();
-    expect(data.edges.length).toEqual(537);
+    expect(data.edges.length).toEqual(520 + TESTING_USERS.length + TESTING_ROLES.length + TESTING_GROUPS.length);
```

Comment:
why did this counter change ? we didn't add any object in the tests in this PR, right ?

Reply:
Actually we need to discuss something: as far as I see the "Administrator" group was not created on platform start, or not created anymore ? So we have added it back. The Administrator role war correctly created, only group is missing.

Reply:
Adding TESTING_USERS.length + TESTING_ROLES.length + TESTING_GROUPS.length is done to be less sensitive to RBAC changes in tests, it has been done previously on other counter in "paginate all the thing" in the platform, so we added that here too, just as "refactor"

Reply:
test coverage added :)